libtransmission/peer-mgr.cc

```diff
@@ -335,6 +335,7 @@ class tr_swarm
             auto& [socket_address, peer_info] = *iter;
             if (peer_info.is_inactive(now))
             {
+                --stats.known_peer_from_count[iter->second.from_first()];
```

Comment:
*(minor nit)*

```suggestion
                --stats.known_peer_from_count[peer_info.from_first()];
```

I know I suggested the way you've written it now, but in hindsight, this is clearer. :P
Either way is harmless, so feel free to ignore if it is too much hassle.